paper-tabs.html

```diff
@@ -655,7 +699,136 @@
+          normalizedLeft *= this.__getRTLPositiveLeft() ? -1 : 1;
+          normalizedLeft += this.__getRTLZeroRight() ? this._tabContainerScrollSize : 0;
+          return normalizedLeft;
+        } else {
+          return scrollLeft;
+        }
+      },
+
+      /**
+       * Converts normalized (LTR-like) `scrollLeft` values to the browser
+       * specific value based on the element's current directionality.
+       */
+      __normalizedLeftToScrollLeft: function(normalizedLeft) {
+        if (this.__getRTL()) {
+          var scrollLeft = normalizedLeft;
+          scrollLeft -= this.__getRTLZeroRight() ? this._tabContainerScrollSize : 0;
+          scrollLeft *= this.__getRTLPositiveLeft() ? -1 : 1;
+          return scrollLeft;
+        } else {
+          return normalizedLeft;
+        }
+      },
+
+      /**
+       * Computes (if needed) and returns `__rtlZeroRight`. See below for a
+       * description of this value.
+       */
+      __getRTLZeroRight: function() {
+        var __rtlBehavior = this.__rtlBehavior;
+        if (__rtlBehavior.__rtlZeroRight === undefined) {
+          this.__checkRTLBehavior();
+        }
+
+        return __rtlBehavior.__rtlZeroRight;
+      },
+
+      /**
+       * Computes (if needed) and returns `__rtlPositiveLeft`. See below for a
+       * description of this value.
+       */
+      __getRTLPositiveLeft: function() {
+        var __rtlBehavior = this.__rtlBehavior;
+        if (__rtlBehavior.__rtlPositiveLeft === undefined) {
+          this.__checkRTLBehavior();
+        }
+
+        return __rtlBehavior.__rtlPositiveLeft;
+      },
+
+      // These properties are nested in this object so that the object reference
+      // is copied to each instance in IE10, where the prototype of an object
+      // can't be changed after it is created.
+      __rtlBehavior: {
+        // True if, when an element has RTL directionality, `scrollLeft` is
+        // zero when the right edge of the scrollable area is aligned with the
+        // right edge of its overflowing content. (As opposed to LTR, where
+        // `scrollLeft` is zero when when the left edge of the scrollable area
+        // is aligned with the left edge of its overflowing content.)
+        // This variable is determined and set in `__checkRTLBehavior` the
+        // first time `__rtlZeroRight` or `__rtlPositiveLeft` is read.
+        __rtlZeroRight: undefined,
+
+        // True if, when an element has RTL directionality, positive changes
+        // to `scrollLeft` cause the scrollable area to show content that is
+        // further to the left. (As opposed to LTR, where positive changes to
+        // `scrollLeft` cause the scrollable area to show overflowing content
+        // that is further to the right.)
+        // This variable is determined and set in `__checkRTLBehavior` the
+        // first time `__rtlZeroRight` or `__rtlPositiveLeft` is read.
+        __rtlPositiveLeft: undefined,
```

Comment:
Removed the `__` here and on the other. Also, maybe a diagram would clear up the difference between these two a bit?